src/Admin/Pool.php

```diff
@@ -331,7 +331,7 @@ public function getAdminGroups(): array
      */
     public function getAdminServiceIds(): array
```

Comment:
maybe we should add a comment for the next major version?

Reply:
Do you mean a comment to rename `getAdminServiceIds()` to `getAdminServiceCodes()`?

Reply:
Indeed, we should add a getAdminServiceCodes method, and deprecate getAdminServiceIds

Reply:
I see. After adding this new method, is it fine to modify other files in this package that calls `getAdminServiceIds()` to `getAdminServiceCodes()` too?

Reply:
I decide to only add the new method `getAdminServiceCodes()` first, with other files unchanged to minimize the scope of this pull request. I can submit another pull request that rename the codes that call the deprecated method. Please see if it is ok, thanks!

Reply:
Thanks. That's ok. Do you want to provide the other PR to fix the deprecated calls ?

Reply:
No problem, I have submitted #7943 for this.